src/main/scala/org/codefeedr/plugins/github/GitHubProtocol.scala

```diff
@@ -0,0 +1,140 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.codefeedr.plugins.github
+
+import java.time.LocalDateTime
+
+import org.codefeedr.pipeline.PipelineItem
+import org.codefeedr.plugins.github.GitHubProtocol.Payload
+import org.json4s.JObject
+
+object GitHubProtocol {
+
+  /**
+    * START /events
+    */
+  case class Event(id: String,
+                   `type`: String,
```

Comment:
If you use `@JsonProperty("type") eventType: String` then we don't need to type `type` every time.